test/unit-test/ota_config.h

```diff
@@ -31,6 +31,28 @@
 #ifndef OTA_CONFIG_H_
 #define OTA_CONFIG_H_
 
-/* Dummy ota_config.h for building the library. */
+#define configENABLED_DATA_PROTOCOLS    ( OTA_DATA_OVER_MQTT | OTA_DATA_OVER_HTTP )
```

Comment:
Why are we defining print functions for a unit test?

Reply:
It's for compiler warnings, without it we will have some unused variable warnings.